doc/contributor/howto-guide-setup-cmake-environment.md

```diff
@@ -60,7 +60,8 @@ to the `cmake` configure step:
 
 ```shell
 cd $HOME/google-cloud-cpp
-cmake -H. -Bcmake-out/home -DCMAKE_TOOLCHAIN_FILE=$HOME/vcpkg/scripts/buildsystems/vcpkg.cmake
+cmake -H. -GNinja -Bcmake-out/home \
```

Comment:
`-H` is an undocumented option. With CMake >= 3.10 (which we now require) we can use the documented `-S .` option (note the space).  Please fix that too.

Reply:
optional: in scripts we would sort these options differently we generally use: `cmake -G Ninja -S . -B cmake-out/home`.

Reply:
Done